climakitae/explore/agnostic.py

```diff
@@ -277,20 +277,21 @@ def create_conversion_function(lookup_tables):
 ##### TASK 2 #####
 
 
-def _get_var_info(variable, downscaling_method):
+def _get_var_info(variable, downscaling_method, wrf_timescale="monthly"):
     """Gets the variable info for the specific variable name and downscaling method"""
     var_desc_df = read_csv_file(variable_descriptions_csv_path)
+    timescale = wrf_timescale if downscaling_method == "Dynamical" else "monthly"
     return var_desc_df[
         (var_desc_df["display_name"] == variable)
-        & (var_desc_df["timescale"].str.contains("monthly"))
+        & (var_desc_df["timescale"].str.contains(timescale))

```

Comment:
Might need to add some error tracing here in terms of "other" options passed... like is daily an option?

Reply:
`daily` has been tested to work, and I will add a validation check to make sure no erroneous strings are being passed in.